e2e-tests/init-deploy/conf/another-name-rs0.yml

```diff
@@ -59,6 +59,19 @@ spec:
         resources:
           requests:
             storage: 1Gi
+    sidecars:
+    - image: busybox
+      command: ["/bin/sh"]
+      args: ["-c", "while true; do echo echo $(date -u) 'test' >> /dev/null; sleep 5;done"]
+      name: my-sidecar-1
+    - image: busybox
+      command: ["/bin/sh"]
+      args: ["-c", "while true; do echo echo $(date -u) 'test' >> /dev/null; sleep 5;done"]
+      name: my-sidecar-1
```

Comment:
@spron-in what operator should do (print error/warning or ignore) if the user has specified 1) container with the name which used for operator containers 2) two containers with the same name?

Reply:
We need to inherit the behavior from Kubernetes for this case. This feature is for experienced users and additional failsafe mechanisms will not be implemented.

Reply:
Let me summarize the behavior:

1. Operator prints an error and skips those containers

2. Operator does nothing and lets kubernetes handle those containers

@spron-in is it right?

Reply:
It is correct